src/query.c

```diff
@@ -1057,15 +1057,46 @@ static IndexIterator *Query_EvalUnionNode(QueryEvalCtx *q, QueryNode *qn) {
 
 typedef IndexIterator **IndexIteratorArray;
 
+static void tag_strtofold(char *str, size_t *len, int caseSensitive) {
```

Comment:
Update function name since it is doing tolower now
```suggestion
static void tag_strtolower(char *str, size_t *len, int caseSensitive) {
```